packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -284,17 +284,18 @@ export default compose(
 		);
 
 		const isDefaultBlock = blockName === getDefaultBlockName();
-		const isEmptyContent = block.attributes.content === '';
+		const isEmptyContent = block?.attributes.content === '';
 		const isExactlyOneBlock = blockOrder.length === 1;
 		const isEmptyDefaultBlock =
 			isExactlyOneBlock && isDefaultBlock && isEmptyContent;
 
-		const selectedBlockClientId = getSelectedBlockClientIds();
-		const selectedBlock = getBlocksByClientId( selectedBlockClientId );
-		const selectedBlockPossibleTransformations = getBlockTransformItems(
-			selectedBlock,
-			rootClientId
-		);
+		const selectedBlockClientId = first( getSelectedBlockClientIds() );
+		const selectedBlock = selectedBlockClientId
```

Comment:
Since the name is singular I changed these values to be object instead of an array.